etc/rc.d/wsdd

```diff
@@ -9,14 +9,11 @@
 
 name=wsdd
 rcvar=wsdd_enable
-wsdd_group=""
+wsdd_group=$(/usr/local/bin/testparm -s --parameter-name workgroup 2>/dev/null)
 
-if which testparm >/dev/null 2>/dev/null; then
-	wsdd_group=$(testparm -s --parameter-name workgroup 2>/dev/null)
-fi
+: ${wsdd_smb_config_file="/usr/local/etc/smb4.conf"}
```

Comment:
why is it `: $...` and not simply `wsdd_smb_config_file=` as below but with the new file name?

Reply:
I updated my shell knowledge 😉 Will merge immediately...